### ISSUE TITLE
Typedoc fails on processing package.json files with BOM prefix

### ISSUE THREAD

**Issue by antongolub** [TypeStrong/typedoc#1192]
This is a very common trouble with paralleljs. For example, [https://github.com/bazelbuild/rules_nodejs/issues/1448](https://github.com/bazelbuild/rules_nodejs/issues/1448)

- [x] I have checked [issues with bug label](https://github.com/TypeStrong/typedoc/labels/bug) and found no duplicates

## Expected Behavior

```
typedoc --readme README.md --tsconfig tsconfig.json src/main
Using TypeScript 3.7.5 from ~/project/node_modules/typescript/lib
Rendering [========================================] 100%

Documentation generated at ~/project/docs
```

## Actual Behavior
```
typedoc --readme README.md --tsconfig tsconfig.json src/main
Error: Could not parse ~/project/node_modules/paralleljs/package.json
```

## Steps to reproduce the bug
1) Add `paralleljs` as a dependency or any other package, that has BOM in its files
2) Run doc generator

## Environment
  - Typedoc version: 0.16.8
  - Node.js version: 12.6.0
  - OS: macOS 10.15.2

**Comment by Gerrit0** [comment]
> `~/project`
Totally unrelated, but we should absolutely do this. Either relative to the user directory or the cwd.

Thanks for the report!

**Comment by Gerrit0** [comment]
Fix released in 0.16.9
